docs/configure-gcp.md

```diff
@@ -56,12 +56,28 @@ This is a setup in the account in which the service shall deploy the actual inst
 2. Navigate to IAM & Admin on GCP.
 3. Click IAM.
 4. Click +ADD.
-5. Fill in the Service Account e-mail under New principals.
-6. Choose the permissions the provisioning app needs (TBC): 
-Roles:
-   - Service Account User
-   - Compute Admin (We are allowing here more permissions than needed)
-7. Click SAVE.
+5. Fill in the Provisioning Service Account pricipal (provisioning-stage@red-hat-hcc.iam.gserviceaccount.com) under New principals.
```

Comment:
Here, I'd not include the exact principal, this is supposed to be deployment agnostic and should be guide to creating both service and Customer principals (or at least that's what we've done for the other cloud providers). It is supposed to be a guide for anyone who deploys the service.
Please use our internal docs https://satellite-services.pages.redhat.com/access/hyperscalers.html to provide accounts used in ConsoleDot deployment.